src/models/inference/classifier_prediction.py

```diff
@@ -30,7 +30,9 @@ class Prediction(BasePrediction):
 
     """
 
```

Comment:
Are we planning to implement prediction for each kind of UDF?

Reply:
This is an old mechanism, outputs for UDFs are replaced with Outcome. I need to check where it's being used and delete if not needed.